package.json

```diff
@@ -7,7 +7,7 @@
   "license": "MIT",
   "repository": "git@github.com:microsoft/react-native-macos.git",
   "engines": {
-    "node": ">=12"
+    "node": ">=14"
```

Comment:
Why is this 14 while the others are 16?

Reply:
As I've explained [here](https://github.com/facebook/react-native/pull/32980#discussion_r794398117):

> it's 14 because on CircleCI we also have a job that tests against the previous LTS (14) hence why the requirement. You can see me bumping that nodeprevlts in this PR